Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/ConceptEntry.java

```diff
@@ -51,7 +51,7 @@
     private String pos;
 
     @SearchField(fieldName = SearchFieldNames.CONCEPT_LIST)
-    @LuceneField(lucenefieldName = LuceneFieldNames.CONCEPT_LIST, isTokenized = false, isMultiple = false)
+    @LuceneField(lucenefieldName = LuceneFieldNames.CONCEPT_LIST, isTokenized = true, isMultiple = false)
```

Comment:
needs to be an exact match. PhraseQueries can be used for that I believe:
https://lucene.apache.org/core/6_3_0/core/org/apache/lucene/search/PhraseQuery.html#PhraseQuery-java.lang.String-java.lang.String...-